Doc/library/ctypes.rst

```diff
@@ -309,10 +309,10 @@ bytes objects are immutable)::
    >>> s = "Hello, World"
    >>> c_s = c_wchar_p(s)
    >>> print(c_s)
-   c_wchar_p('Hello, World')
+   c_wchar_p(140018365411392)
    >>> c_s.value = "Hi, there"
    >>> print(c_s)
```

Comment:
Same as above. Without using `.value` the whole example doesn't do a good job on explaining the subject.